api/v1beta1/verticadb_types.go

```diff
@@ -890,6 +890,7 @@ func MakeVDB() *VerticaDB {
 				DataPath:  "/data",
 				DepotPath: "/depot",
 			},
+			KSafety:    KSafety1,
```

Comment:
Why adding this now?

Reply:
I used the `MakeVDB()` function in the webhook UT.  When I tried to test the successful case, where the requeue times were valid, it still complained that the kSafety was not valid.  We just never set it properly, so any webhook test will complain about it.